conan/tools/system/pip_manager.py

```diff
@@ -23,6 +24,24 @@ def generate(self):
         env.prepend_path("PATH", self.bin_dir)
         env.vars(self._conanfile).save_script(self.env_name)
 
+    @staticmethod
+    def _default_python():
+        default_python = shutil.which('python') if platform.system() == "Windows" else shutil.which('python3')
+        return os.path.realpath(default_python) if default_python else None
+
+    def _create_venv(self):
+        python_interpreter = self._conanfile.conf.get("tools.system.pipenv:python_interpreter") or self._default_python()
+        if not python_interpreter:
+            raise ConanException("PipEnv could not find a Python executable path."
+                                 "Please set 'tools.system.pipenv:python_interpreter' to '</your/python/full/path>' "
+                                 "in the [conf] section of the profile, or in the command line using "
+                                 "'-c tools.system.pipenv:python_interpreter=</your/python/full/path>'")
```

Comment:
I think the message should say two things:
- Please, install Python system-wide
- Or set the "tools.system.pipenv:python_interpreter" conf to the full path of a Python executable (net necessary to say that it can go in the [conf] or in the command line)